### PR TITLE
Writing asked functions by Axios API

### PR DESCRIPTION
**Pull request by SeyedMojtaba1**
related #25

### CODE REVIEW

File: Seyed/API calls/main.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+interface Query{
+    id?: number,
+    userId?: number,
+    title?: string,
+    body?: string
+}
+
+async function searchForPosts(input: Query) {
+    try{
+        const response = await axios.get(`https://jsonplaceholder.typicode.com/posts`)
```
Comment (fatemeh-ra) [inline Seyed/API calls/main.ts:62]: use the search route

**Review by fatemeh-ra — CHANGES_REQUESTED** [review]

File: Seyed/API calls/main.ts (hunk 1 above)
Comment (fatemeh-ra) [inline Seyed/API calls/main.ts:62]: write the search function using provided query parameters

**Review by fatemeh-ra — CHANGES_REQUESTED** [review]

File: Seyed/API calls/main.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
+import axios from "axios"
+
+async function display10Posts() {
+    try{
+        const response = await axios.get('https://jsonplaceholder.typicode.com/posts')
```
[the remark this answers is not in the document]
Comment (SeyedMojtaba1) [reply Seyed/API calls/main.ts:5]: I fixed it and used from query parameters.

[the remark this answers is not in the document]
Comment (SeyedMojtaba1) [reply Seyed/API calls/main.ts:5]: I fixed it.

File: Seyed/API calls/main.ts (hunk 1 above)
Comment (SeyedMojtaba1) [reply Seyed/API calls/main.ts:62]: I fixed it based on query filters parameters.

File: Seyed/API calls/main.ts [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+interface Query{
+    id?: number,
+    userId?: number,
+    title?: string,
+    body?: string
+}
+
+async function searchForPosts(input: Query) {
+    try{
+        const response = await axios.get(`https://jsonplaceholder.typicode.com/posts/?id=${input.id}&?userId=${input.userId}&?body=${input.body}&?title=${input.title}`)
```
Comment (fatemeh-ra) [inline Seyed/API calls/main.ts:77]: what if some of them were not defined? the `Query` object properties are not required.

**Review by fatemeh-ra — CHANGES_REQUESTED** [review]

**Review by fatemeh-ra — APPROVED** [review]

**Pull request merged**